community/kernel/src/docs/dev/batchinsert.asciidoc

```diff
@@ -12,8 +12,8 @@ Be aware of the following points when using batch insertion:
 * The intended use is for initial import of data but you can use it on an existing database if the existing database is shutdown first.
 * Batch insertion is _not thread safe._
 * Batch insertion is _non-transactional._
-* Batch insertion will re-populate all existing indexes and indexes created during batch insertion
-  on shutdown.
+* Batch insertion is _not enforcing constraints_ on the inserted data.
+* Batch insertion will re-populate all existing indexes and indexes created during batch insertion on shutdown.
```

Comment:
Constraints need indexes to work, which are only available after insertion. Makes sense.
But this (for me) instantly raises the question what happens when I use the BatchInserter on an existing DB with existing constraints and batch insert data that violates a constraint. I end up with an inconsistent DB, right? One that claims a constraint is enforced when there's data that does not follow that constraint.
I'm aware that BatchInserter is a _know what you're doing or don't do it_-deal, but it seems too easy to accidentially produce inconsistent data. 
Maybe add a line stating that dropping and re-adding constraints after batch insertions ensures all constraints are still meet (or the second step fails)? Hm, that could be an (optional) final step during shutdown after building indexes maybe?